tools/pd-tso-bench/main.go

```diff
@@ -99,7 +100,7 @@ func bench(mainCtx context.Context) {
 	ctx, cancel := context.WithCancel(mainCtx)
 	// To avoid the first time high latency.
 	for idx, pdCli := range pdClients {
-		_, _, err := pdCli.GetTS(ctx)
```

Comment:
It seems we don't need GetTS anymore?

Reply:
Yes, it's equivalent to getting a local TSO from the "global". But considering the backward compatibility, maybe we should keep it in the PD client?

Reply:
How about marking it as deprecated?